src/gui/tray/NCButtonContents.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
```

Comment:
should we not have `.15` everywhere ?
probably easier to predict the result in case the components have been modified

Reply:
Yes, upgraded the minor versions